.stylelintrc

```diff
@@ -0,0 +1,147 @@
+{
+  "rules": {
+    "at-rule-empty-line-before": [ "always", {
+      except: [
+        "blockless-after-same-name-blockless",
+        "first-nested",
```

Comment:
cool, okay. and do we have a linter rule that ensures extends/includes and other at keywords go first in a block, presuming that's something we want?